### ISSUE TITLE
F.lux support

### ISSUE THREAD

**Issue by Weechit** [psieg/Lightpack#434]
Hey there, having the latest version of Pristmatik and f.lux - LED values do adjust but only so subtle (whenever f.lux shifts towards warm colours). Is it a known thingy?

Also, Windows 10 Night Light option does work OK

**Comment by Arnseb** [comment]
Same for me. Running adalight on an Arduino Nano, but doesn't work with Night Light either (although my Night Light mode is bugged).

**Comment by t0stiman** [comment]
enabling the "apply screen color modifications" with f.lux does not do anything on my system

**Comment by psieg** [comment]
I tested several combinations and got weird results:
Prismatik 5.11.2.27 + Flux 4 = not detected
Prismatik 5.11.2.27 + Flux 3 = not detected
Prismatik 5.11.2.19 + Flux 4 = not detected
Prismatik 5.11.2.19 + Flux 3 = detected

I also tested another random tool that sets the gamma ramp (which is what this feature uses) and only 5.11.2.19 would detect that. Yet it doesn't detect Flux 4. Unfortunately I can't remember whether this ever worked with Flux 3...

Either both Prismatik *and* F.lux have changed, or it's something in Windows that changed :/

**Comment by zomfg** [comment]
I tried .27 / 4 in windows sandbox, it seems to work at least while using the big slider in f.lux (f.lux thinks it's in remote desktop when in winsandbox and disables normal mode)
also Night Light needs to be disabled for Prismatik to use f.lux gamma tables iirc, Night Light could be set to 6500K and look inactive to the eye but that would make Prismatik skip f.lux

**Comment by psieg** [comment]
Good point about the precedence. With NightLight disabled (not just the quick button in the notification center, but the schedule in windows settings) then

5.11.2.19 + Flux 3 = detected

With Flux 4 I still don't get anything, even when turning "Use GPU for better quality with very warm colors" is off. I don't know what it uses to set the color (and how to read that) or if it's possible to force it to use the gamma ramp mode

**Comment by t0stiman** [comment]
I can confirm that F.lux 3 works, you can download it here:
https://forum.justgetflux.com/topic/3984/downloading-old-copies-of-f-lux
